### DISCUSSION TITLE
Annotations List Panel should render HTML Annotations

### DISCUSSION THREAD

**Discussion by jim-stroup** [grafana/grafana#40793] [Dashboards, Panels & Visualizations]
**What happened**:
The Annotations List Panel does not render HTML formatted annotations.

**What you expected to happen**:
I expect the text to be HTML formatted (e.g. permit hyperlinks, thumbnails, etc.) 

**How to reproduce it (as minimally and precisely as possible)**:
Insert a simple HTML formatted annotation via Grafana's HTTP API for annotations. Then try to view that same annotation in Grafana's Annotation List panel.

**Anything else we need to know?**:
See screenshots for reference. ***A great short term solution would be to enable titles for annotations and prioritize displaying the title of this annotation over the description text - this way you could have HTML formatted hover-over tool-tips with a simple/functional annotations panel.***

![annotations_issue_2](https://user-images.githubusercontent.com/64607347/88289075-b1630600-ccc2-11ea-91ab-1da1fda7a7e3.jpg)

![annotations_issue](https://user-images.githubusercontent.com/64607347/88289053-aa3bf800-ccc2-11ea-8e6d-3b5b8f933708.jpg)

**Environment**:
- Grafana version:  v7.1
- Data source type & version:  N/A
- OS Grafana is installed on:  Ubuntu 18.04.4 LTS
- User OS & Browser:  Windows 10/Edge Chromium
- Grafana plugins:  Annotation List Panel
- Others: N/A

Original Issue raised while the panel was community developed:  #26550

**Comment by jim-stroup** [comment]
This is a re-attack since Grafana now officially supports the annotations panel and it is no longer community-driven.

**Comment by hugohaggmark** [comment]
I think the annotation list is virtualized. This currently means that each item needs to have the exact same height, maybe this can be changed. Marking this as a feature-request and I'm going to convert this to a Github discussion instead. Thank you for bringing this up @jim-stroup.

**Reply by jim-stroup** [reply]
Thanks for the quick response @hugohaggmark !

I've not seen where it is a fixed length - I tried to put plain text in the descriptor (first followed by HTML, second attempt was  followed by a long string of spaces) hoping that I could use its fixed length as a workaround. It resized to show the full content both times.

At the core of this issue: Grafana renders annotations received from its API differently across two of its officially supported components (e.g. time-series panel and annotations panel). That's why I submitted this issue as a bug, not a feature request. 

Do you know of a good workaround in the meantime? The annotations panel is a powerful tool that we really want to leverage but cant because our annotations are html formatted.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
